ln1_f4.py

```diff
@@ -145,13 +138,14 @@ def get_raw_data(command): #функція виконує консольну к
             </table>
         </body>
     </html>
-    """.format(str(cpu_total),str(cpu_user),str(cpu_sys),str(cpu_idle),str(mem_total),str(mem_used),str(mem_free),str(mem_cached),str(hdd_total),str(hdd_used),str(hdd_free))
+    """.format(str(cpu_total), str(cpu_user), str(cpu_sys), str(cpu_idle), str(mem_total), str(mem_used), str(mem_free),
```

Comment:
probably you don't need call `str` in format method. It transform to str automatically